AkismetPlugin.inc.php

```diff
@@ -177,7 +177,8 @@ function checkAkismet($hookName, $args) {
 		$iso639_1 = array();
 		foreach (array_keys($locales) as $locale) {
 			// Our locale names are of the form ISO639-1 + "_" + ISO3166-1 
-			$iso639_1[] = array_pop(explode('_', $locale, 1));
+			$localeList=explode('_', $locale);
```

Comment:
We now have access to a library which should be able to do this work for us:
https://github.com/pkp/pkp-lib/issues/9278

https://github.com/pkp/pkp-lib/blob/stable-3_4_0/classes/i18n/LocaleConversion.php